py3-azure-storage-blob.yaml

```diff
@@ -0,0 +1,49 @@
+# Generated from https://pypi.org/project/azure-storage-blob/
+package:
+  name: py3-azure-storage-blob
+  version: 12.19.0
+  epoch: 0
+  description: Microsoft Azure Blob Storage Client Library for Python
+  copyright:
+    - license: MIT License
```

Comment:
This should just be 'MIT', which is the spdx short identifier:
https://spdx.org/licenses/MIT.html